### ISSUE TITLE
Fresh Svelte Storybook project is not loading, printing garbage to CLI

### ISSUE THREAD

**Issue by mattfysh** [storybookjs/storybook#17181]
**Describe the bug**

1. Garbage characters are being printed when attempting to `npm run storybook`
1. Page does not load, stuck with infinite loading loop
1. Only way forward is to CTRL+C terminate the storybook process

![image](https://user-images.githubusercontent.com/470258/148708906-ba77d20b-0337-48ba-89a6-2c92c54aa586.png)

**To Reproduce**

```sh
# Create a new Svelte app
npx degit sveltejs/template my-svelte-project
cd my-svelte-project
npm install

# Install Storybook
npx sb init
# This also does not work:
# npx sb@next init

# Run Storybook
npm run storybook
```

**System**

```yaml
  System:
    OS: macOS 12.1
    CPU: (8) x64 Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
  Binaries:
    Node: 17.3.0 - /usr/local/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 8.3.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 97.0.4692.71
    Safari: 15.2
```

**Comment by mkohlmyr** [comment]
Same thing with storybook for react.

Mac 10.15.6, Darwin Kernel Version 19.6.0
2.3 GHz Dual-Core Intel Core i5
8 GB 2133 MHz LPDDR3

**Comment by MassimoFortarezza** [comment]
Same thing. Working till yesterday at about 8pm. Then suddendly stopped working :(

**Comment by mattfysh** [comment]
I wonder if this is the colors.js infinite loop issue?

**Comment by mattfysh** [comment]
It looks like it was caused by the colors package, v1.4.2 introduced an infinite loop as a protest by the module author

To fix: Update all your storybook packages to use the `next` tag

**Comment by shilman** [comment]
@mattfysh I've also patched this back to `6.4.10`

**Comment by mattfysh** [comment]
Thanks @shilman for the quick turnaround :) no need to apologise!
